test/jdk/jdk/classfile/VerifierSelfTest.java

```diff
@@ -51,41 +47,10 @@ void testVerify() throws IOException {
                     .flatMap(p -> p)
                     .filter(p -> Files.isRegularFile(p) && p.toString().endsWith(".class")).forEach(path -> {
                         try {
-                            ClassFile.of().parse(path).verify(null);
+                            ClassFile.of().verify(path);
                         } catch (IOException e) {
                             throw new AssertionError(e);
                         }
                     });
     }
-
-    @Test
-    void testFailedDump() throws IOException {
```

Comment:
Why is this removed?

Reply:
Dump (print) of the classfile to an optional log (Consumer<String> argument) has been removed from the API.
It was a relic from early phase of the ClassFile API development and it has no use except for this test.
This functionality can be replaced by explicit use of ClassPrinter.

Reply:
I'll rename the test to "testFailed" and keep it without the "dump" park.